### ISSUE TITLE
Fix Travis build

### ISSUE THREAD

**Issue by danielparton** [choderalab/TargetExplorer#24]
Last PR merge broke Travis:
https://travis-ci.org/choderalab/TargetExplorer/builds/108440832

Traceback:

```
...
installing: wheel-0.26.0-py27_1 ...
installing: setuptools-18.8.1-py27_0 ...
Python 2.7.11 :: Continuum Analytics, Inc.
creating default environment...
installation finished.
sudo: must be setuid root
devtools/travis-ci/install.sh: line 15: conda: command not found
devtools/travis-ci/install.sh: line 16: activate: No such file or directory
devtools/travis-ci/install.sh: line 17: conda: command not found
devtools/travis-ci/install.sh: line 18: conda: command not found
```

I'm guessing this is due to a change in Travis. TargetExplorer hasn't been updated for a while, and Travis may have changed in the meantime.
So hopefully should be a simple fix.

**Comment by jchodera** [comment]
It's probably just because travis hasn't been run recently.  I can fix that.

**Comment by jchodera** [comment]
This is a good opportunity to point out [the Travis CI cron trigger](http://traviscron.pythonanywhere.com).

**Comment by steven-albanese** [comment]
@jchodera I think I've got Travis build fixed in PR #25 if you want to take a look and merge it

**Comment by steven-albanese** [comment]
I think this has been taken care of! I'm going to close this.
